client/src/components/OfflineWarning/OfflineWarning.js

```diff
@@ -3,17 +3,32 @@ import PropTypes from 'prop-types';
 
 import './offline-warning.css';
 
+const delayInMilliSeconds = 5000;
+var id;
```

Comment:
Can we use `let` here`?
```suggestion
let id;
```

Reply:
Yeah

Reply:
@RandellDawson i apply the changes